### PR TITLE
MRSD '23 Team G: Docker for PyTorch

### PR DESCRIPTION
**Pull request by PallaviMadhukar**

### CODE REVIEW

File: wiki/tools/docker-for-pytorch.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+ 
+But before we go ahead, make sure that you have NVIDIA GPU drivers installed. You can check if the drivers are installed by running the following command. If this fails, it means that the drivers are not installed and you first have to do that before proceeding ahead.
+
+```properties
+nvidia-smi
+```
+ 
+For running the NVIDIA Container Toolkit, we can simply pull the NVIDIA Container Toolkit image at the top of our Dockerfile like so,
+ 
+<em><strong>
```
Comment (smbryan) [inline wiki/tools/docker-for-pytorch.md:30]: Please don't mix HTML in with the Markdown; Jekyll doesn't handle it how you'd expect. 

CC: @nevalsar

**Review by smbryan — CHANGES_REQUESTED** [review]
This is really close to be ready to merge. Please drop the HTML and use the Markdown equivalent.

**Review by nevalsar — CHANGES_REQUESTED** [review]
(See earlier review comments)

**Comment by nevalsar** [comment]
Contributor unresponsive - will fix and merge these changes in a separate MR.

**Pull request closed without merging**